packages/esm-patient-tests-app/src/test-results/panel-view/panel-view.component.tsx

```diff
@@ -29,32 +25,39 @@ interface PanelViewHeaderProps {
   totalSearchResults: number;
 }
 
-const PanelView: React.FC<PanelViewProps> = ({ expanded, testUuid, basePath, type, patientUuid }) => {
+const PanelView: React.FC<PanelViewProps> = ({ expanded, patientUuid }) => {
   const { t } = useTranslation();
   const layout = useLayoutType();
   const isTablet = layout === 'tablet';
-  const trendlineView = testUuid && type === 'trendline';
```

Comment:
This must have been a feature a long time ago. But this 'type' comes from a URL parameter set by `navigateToTrendline`, and nothing calls that function anymore.